### ISSUE TITLE
kxstudio-meta-audio-applications won't install.

### ISSUE THREAD

**Issue by tvvoty** [KXStudio/Repository#267]
* I figured out what was wrong while I was writing this, but still decided to post, in case anyone else have it, you can close it right away I guess.

When I try to mark kxstudio-meta-audio-applications package for installation in muon, I get this error:
The "kxstudio-meta-audio-applications" package could not be marked for installation or upgrade:
Depends: kxstudio-meta-audio-applications, but it is not installable
It looks like it depends on itself somehow?

OS: Kubuntu 21.04

If I try to install it through terminal with "sudo apt-get install kxstudio-meta-audio-applications" I get this:
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 studio-controls : Conflicts: cadence but 5:0.9.1 is to be installed
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
```

Solution:
kx repos don't like studio-controls that comes with ubuntustudio-installer (I thought it wouldn't matter since no ubuntu studio packages were installed, but I was wrong). So don't install it and if you have - delete it.

**Comment by tvvoty** [comment]
I figured out what was wrong while I was writing the issue, but still decided to post, in case anyone else have it.

Solution:
kx repos don't like studio-controls that comes with ubuntustudio-installer (I thought it wouldn't matter since no ubuntu studio packages were installed, but I was wrong). So don't install it and if you have - delete it.

**Comment by falkTX** [comment]
dont close this is a valid issue.
studio-controls conflicts with cadence, but one way around this is to make the metapackage require one or the other.

**Comment by tvvoty** [comment]
Oh, ok then!

**Comment by falkTX** [comment]
Fixed in 94a0ec34f2a4dce6f355a610b1740db25d1f4bb7, update will appear in the repos soon
